src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ControlTests.cs

```diff
@@ -721,6 +721,282 @@ public void Control_CreateParams_GetVisible_ReturnsExpected(bool visible, int ex
             Assert.False(control.IsHandleCreated);
         }
 
+        [WinFormsTheory]
+        [InlineData(RightToLeft.No)]
+        [InlineData(RightToLeft.Yes)]
+        public void Control_GetNextItem_Buttons_CycleForwardExpected(RightToLeft rightToLeft)
+        {
+            using SubControl control = new()
+            {
+                RightToLeft = rightToLeft
+            };
+            using Button button1 = new();
+            using Button button2 = new();
+            using Button button3 = new();
+            control.Controls.AddRange(new Button[] { button1, button2, button3});
+            Control actual = control.GetNextControl(button1, forward: true);
+            Assert.Equal(button2, actual);
+            actual = control.GetNextControl(actual, forward: true);
+            Assert.Equal(button3, actual);
+            actual = control.GetNextControl(actual, forward: true);
+            Assert.Null(actual);
```

Comment:
```suggestion
            Control nextControl1 = control.GetNextControl(button1, forward: true);
            Control nextControl2 = control.GetNextControl(button2, forward: true);
            Control nextControl3 = control.GetNextControl(button2, forward: true);
            
            Assert.Equal(button2, nextControl1);
            Assert.Equal(button3, nextControl2);
            Assert.Null(nextControl3);
```
I would suggest separating the code with getting and validating the results. I would also update the tests below.